docs/develop/best-practices/accessibility.md

```diff
@@ -26,9 +26,449 @@ The technology is standards-compliant and performant.
+    Comes with MacOS (just check your Launchpad!).
+- [NVDA](https://www.nvaccess.org/download/) for Windows (free download).
+
+
+## Best practices
+
+For InvenioRDM to be accessible for everyone, there are a few things to consider when developing the frontend. For instance, all interactive elements should be accessible by keyboard, and the HTML should be optimized for assistive technologies.
+
+
+### Use meaningful markup
+{==__Semantic HTML should always be used, unless the element is only for styling purposes.__==}
+
+Using Semantic HTML is probably one of the most important things we can do to ensure accessiblity. Semantic HTML provides context and meaning and to assistive technologies, as well as default keyboard accessibility for interactive elements.
+
+When using Semantic UI React, please be aware that the components are not always converted to an element with semantic meaning. In most cases, you must specify the element by adding `as={tagName}`. Pay attention to this by inspecting the elements in the browser.
+
+✅ __DO__
+```html
+<!-- 
+Use semantic HTML tags unless the only purpose 
+of the element is styling 
+-->
+
+<header class="ui fluid container">
+    <div class="ui container">
+        <nav></nav>
+    </div>
+</header>
+
+<main class="ui container">
+    <h1>My page</h1>
+    <section>
+        <h2>My section content</h2>
+        <ul>
+            <li>List item 1</li>
+            <li>List item 2</li>
+        </ul>
+        <button>Button text</button>
+    </section>
+</main>
+
+<footer class="ui fluid container">
+    Footer content
+</footer>
+```
+
+_With Semantic UI React:_
+
+```js
+// Set the "as"-prop to the semantic HTML element you want
+
+<Container fluid as="header">
+    <Container>
+        <Menu as="nav"></Menu>
+    </Container>
+</Container>
+
+<Container as="main">
+    <Header as="h1">My page</Header>
+    <Container as="section">
+        <Header as="h2">My section content</Header>
+        <List as="ul">
+            <List.Item as="li">List item 1</List.Item>
+            <List.Item as="li">List item 2</List.Item>
+        </List>
+        <Button>Button text</Button>
+    </Container>
+</Container>
+
+<Container fluid as="footer">
+    Footer content
+</Container>
+```
+
+❌ __DON'T__
+```html
+<!-- Don't use divs or spans in place of semantic elements -->
+
+<div class="ui fluid container">
+    <div class="ui container">
+        <div class="navigation"></div>
+    </div>
+</div>
+
+<div class="main">
+    <div class="page-header">My page</div>
+    <div class="section">
+        <div class="section-header">My section content</div>
+        <div class="list">
+            <div class="list-item">List item 1</div>
+            <div class="list-item">List item 2</div>
+        </div>
+        <span class="button">Button text</span>
+    </div>
+</div>
+
+<div class="ui fluid container">
+    Footer content
+</div>
+```
```

Comment:
In the case where you cannot avoid using a div or another non-semantic element, but wish to provide semantic meaning to that element, you can use roles.   https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles